### PR TITLE
Feature/app 511 list item address

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

This PR adds the ListItem->Address component to the UI library. 

Task: [511](https://aragonassociation.atlassian.net/browse/APP-511?atlOrigin=eyJpIjoiMjMxNGZkZDJhYmFhNGJjOTkwZWEyZDc1ODE2NGQzZDciLCJwIjoiaiJ9)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have tested my code on the test network.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.

### CODE REVIEW

File: packages/ui-components/src/components/listItem/address.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            {tokenInfo.amount} {tokenInfo.symbol} ({tokenInfo.percentage}%)
+          </p>
+        )}
+        <IconLinkExternal />
+      </LeftContent>
+    </Container>
+  );
+};
+
+const Container = styled.button.attrs(() => {
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/listItem/address.tsx:44]: There is box-shadow and a border (sometimes) on the variant states

**Review by Fabricevladimir — CHANGES_REQUESTED** [review]
Thanks, looking good. Left a single comment

**Review by RakeshUP — APPROVED** [review]
Good to go after the minor css changes!

**Review by sepehr2github — COMMENTED** [review]
Thanks, Vali, Just a tiny thing. The external link button isn't centered. may I ask you to take a look? 

Screen Shot 2022-06-06 at 11 07 08 AM

**Comment by WissenIstNacht** [comment]
@sepehr2github, just download Firefox and problem solved! 😋

Jokes aside, Chrome seems to need the Container to have `items-center` as well, which I added now. Thanks for catching this! :)

**Review by Fabricevladimir — APPROVED** [review]
Good to go

**Review by sepehr2github — APPROVED** [review]
Thanks, for the change.

**Pull request merged**